packages/relay/src/lib/services/hapiService/hapiService.ts

```diff
@@ -250,6 +246,19 @@ export default class HAPIService {
     return client;
   }
 
+  private initPrivateKey(operatorMainKey: string): PrivateKey {
+    switch (process.env.OPERATOR_KEY_FORMAT) {
+      case 'DER':
+        return PrivateKey.fromStringDer(operatorMainKey);
+      case 'HEX_ED25519':
+        return PrivateKey.fromStringED25519(operatorMainKey);
+      case 'HEX_ECDSA':
+        return PrivateKey.fromStringECDSA(operatorMainKey);
+      default:
```

Comment:
we should either throw an error if the `OPERATOR_KEY_FORMAT` is not `DER`, `HEX_ED25519`, or `HEX_ECDSA`, or if we want to accept arbitrary values of `OPERATOR_KEY_FORMAT`, then we can remove the case 'DER` block as it's gonna fall into the default block anyway.

Reply:
I agree that it should use the `default` when no format is explicitly specified, otherwise it would be better to throw an error. of `INVALID_PRIVATE_KEY_FORMAT` or something of sorts. that would also resolve your previous comment on documentation being wrong @quiet-node. 👍

Reply:
Done.